views/js/controller/items/edit.js

```diff
@@ -37,12 +37,20 @@ function($, __, module, actions, lock, section){
         start() {
             const config = module.config();
 
-            const previewAction = actions.getBy('item-preview');
-            const authoringAction = actions.getBy('item-authoring');
-
-            if(previewAction){
-                previewAction.state.disabled = !config.isPreviewEnabled;
+            const getPreviewId = idx => `item-preview${idx ? `-${idx}` : ''}`;
+            const previewActions = []
```

Comment:
Not sure, but at least for me the new linter complains about it on portal, etc. There are other places here also missing `;`

```suggestion
            const previewActions = [];
```

Reply:
Good catch. I had disabled auto-formatting, and forgot to restore it... I will update the code